junghum21/ArchStudy/app/src/main/java/com/jake/archstudy/data/source/UpbitRemoteDataSource.kt

```diff
@@ -0,0 +1,57 @@
+package com.jake.archstudy.data.source
+
+import com.jake.archstudy.network.response.MarketResponse
+import com.jake.archstudy.network.response.TickerResponse
+import com.jake.archstudy.network.service.UpbitService
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class UpbitRemoteDataSource(private val upbitService: UpbitService) : UpbitDataSource {
+
+    override fun getMarketAll(
+        success: (List<MarketResponse>) -> Unit,
+        failure: (Throwable) -> Unit
+    ) {
+        upbitService.getMarketAll()
+            .enqueue(object : Callback<List<MarketResponse>?> {
+                override fun onFailure(call: Call<List<MarketResponse>?>, t: Throwable) {
```

Comment:
onResponse와 동일하게 파라미터를 정리해주세요.
```suggestion
                override fun onFailure(
                        call: Call<List<MarketResponse>?>,
                        t: Throwable
                 ) {
```

Reply:
https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/188/commits/2c940ccc4f5e47087980f273660c262bf0859b1b
반영완료